guide/interactions/replying-to-slash-commands.md

```diff
@@ -278,6 +280,59 @@ console.log([string, integer, boolean, user, member, channel, role, mentionable]
 If you want the Snowflake of a structure instead, grab the option via `get()` and access the Snowflake via the `value` property. Note that you should use `const { value: name } = ...` here to [destructure and rename](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) the value obtained from the <DocsLink path="typedef/CommandInteractionOption">`CommandInteractionOption`</DocsLink> structure to avoid identifier name conflicts.
 :::
 
+### Subcommands
+
+If you have a command that contains subcommands, you can parse them in a very similar way as to the above examples.
+Let's say your command looks like this:
+
+```js {4-22}
+const data = {
+	name: 'info',
+	description: 'Get info about a user or the server!',
+	options: [
+		{
+			name: 'user',
+			description: 'Info about a user',
+			type: 'SUB_COMMAND',
+			options: [
+				{
+					name: 'target',
+					description: 'The user',
+					type: 'USER',
+				},
+			],
+		},
+		{
+			name: 'server',
+			description: 'Info about the server',
+			type: 'SUB_COMMAND',
+		},
+	],
+};
+```
+
+The following snippet details the logic needed to parse the subcommands and respond accordingly using the `CommandInteractionOptionResolver#getSubcommand()` method:
+
+```js {5-20}
+client.on('interactionCreate', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'info') {
+		if (interaction.options.getSubcommand() === 'user') {
```

Comment:
```suggestion
      const subCommand = interaction.options.getSubCommand();
		if (subCommand === 'user') {
```
and below use the `subCommand` variable

Reply:
unsure if that's a cleaner approach, imo the direct comparison on the resolver method is clear and concise

Reply:
this suggestion is incorrect - it's `.getSubcommand()`, not `.getSubCommand()`